src/OperationalContext/OperationalContext.tsx

```diff
@@ -16,10 +10,11 @@ export interface Props {
  * This component simply wraps OperationalContext in order to allow styleguidist to pick up on
  * it and display it in the documentation page.
  */
-const OperationalContext: React.SFC<Props> = props => (
-  <OperationalContextOriginal>{props.children}</OperationalContextOriginal>
-)
+const OperationalContext: React.SFC<Props> = props => {
+  const ctx = useOperationalContext()
+  return <React.Fragment>{props.children({ ...ctx })}</React.Fragment>
```

Comment:
Do we need a fragment here? 🤔

Reply:
Yes, It gives a typing error otherwise for directly returning children.

Reply:
Have you tried 

```suggestion
  return <>{props.children({ ...ctx })}</>
```

🤔